src/main/java/com/code_intelligence/jazzer/mutation/combinator/MutatorCombinators.java

```diff
@@ -174,6 +174,57 @@ public R detach(R value) {
     };
   }
 
+  public static <T, R> SerializingMutator<R> mutateThenMapToImmutable(
```

Comment:
Why is this function needed? The overload that already exists should emit both the `toString()` of the mutator it uses under the hood as well as the target type.

Reply:
The idea was to ultimately pass the concrete enum type. We might still come back to this if we cannot get it in any other way!